core/frontend/src/components/health/GpsTrayMenu.vue

```diff
@@ -104,6 +94,7 @@ export default Vue.extend({
 
       this.last_message_date = new Date()
       this.gps_raw_int = message?.message as GpsRawInt
+      this.gps_connected = this.gps_raw_int?.fix_type?.type !== GpsFixType.GPS_FIX_TYPE_NO_GPS
```

Comment:
We should still add a time validation for the case where mavlink data is not received